integration-libs/punchout/components/punchout-requisition/punchout-requisition.component.ts

```diff
@@ -61,12 +61,6 @@ export class PunchoutRequisitionComponent implements OnInit, OnDestroy {
     );
 
   ngOnInit(): void {
-    this.globalMessageService.add(
-      {
-        key: 'punchout.redirectToProcurementSystem',
-      },
-      GlobalMessageType.MSG_TYPE_INFO
-    );
     this.punchoutFormGroup = this.formBuilder.group({
       [this.FORM_CONTROL_NAME.ORDER]: [''],
     });
```

Comment:
do we really need that?

``` 
 ngOnDestroy(): void {
    this.globalMessageService.remove(GlobalMessageType.MSG_TYPE_INFO);
  }
```

Reply:
i aggree, no need to remove it.
use case 1: when success, we leave spartacus
use case2: if failure, land in login page, info message would close after few sec automatically.